README.md

```diff
@@ -253,6 +257,25 @@ BenchmarkReconstruct50x20x1M-8       1364.35      4189.79      3.07x
 BenchmarkReconstruct10x4x16M-8       1484.35      5779.53      3.89x
 ```
 
+# Performance on AVX512
+
+The performance on AVX512 has been accelerated for Intel CPUs. This gives speedups on a per-core basis of up to 4x compared to AVX2 as can be seen in the following table:
+
+```
+$ benchcmp avx2.txt avx512.txt
+benchmark                      AVX2 MB/s    AVX512 MB/s   speedup
+BenchmarkEncode8x8x1M-72       1681.35      4125.64       2.45x
+BenchmarkEncode8x4x8M-72       1529.36      5507.97       3.60x
+BenchmarkEncode8x8x8M-72        791.16      2952.29       3.73x
+BenchmarkEncode8x8x32M-72       573.26      2168.61       3.78x
+BenchmarkEncode12x4x12M-72     1234.41      4912.37       3.98x
+BenchmarkEncode16x4x16M-72     1189.59      5138.01       4.32x
+BenchmarkEncode24x8x24M-72      690.68      2583.70       3.74x
+BenchmarkEncode24x8x48M-72      674.20      2643.31       3.92x
```

Comment:
Could you check odd number of shards, since you have some special handling of that?

Reply:
Here they are for a single core (on a different server):
```
BenchmarkEncodeAvx512_8x5x8M-2               100          19237008 ns/op        3488.53 MB/s
BenchmarkEncodeAvx512_8x6x8M-2               100          16704100 ns/op        4017.51 MB/s
BenchmarkEncodeAvx512_8x7x8M-2                50          27117907 ns/op        2474.71 MB/s
BenchmarkEncodeAvx512_8x8x8M-2               100          19023546 ns/op        3527.67 MB/s
BenchmarkEncodeAvx512_8x9x8M-2                50          29198817 ns/op        2298.34 MB/s
BenchmarkEncodeAvx512_8x10x8M-2               50          26270455 ns/op        2554.54 MB/s
BenchmarkEncodeAvx512_8x11x8M-2               50          36617217 ns/op        1832.71 MB/s
```
"Slowest" are 9 (=x4+x4+1) and 11 (=x4+x4+2x+1) respectively which includes last output row which is non-AVX512 accelerated.

Reply:
Quite a big slowdown. Well, that will be an optimization for another day :)